### ISSUE TITLE
Yanked on crates.io?

### ISSUE THREAD

**Issue by evgeny-yashin** [idubrov/dfp-rs#1]
Hi man,
I wonder what is the reason for the lib to be yanked on the crates.io?
https://docs.rs/dfp/0.1.0/dfp/

>  This release has been yanked

What is the actual status of the lib, can it be used?

**Comment by idubrov** [comment]
No, it is not usable. It is not complete. I started doing bindings to Intel library, but then switched to re-implementing everything in Rust. And then abandoned it 🤷🏻‍♂️
